Formula/timewarrior.rb

```diff
@@ -0,0 +1,19 @@
+class Timewarrior < Formula
+  desc "Command-line time tracking application"
+  homepage "https://taskwarrior.org/docs/timewarrior/"
+  url "http://taskwarrior.org/download/timew-1.0.0.tar.gz"
+  sha256 "ac027910e1e8365bdd218a8b42389b26d017d38d3c96516c408db6d5a44e0bb5"
+  head "https://git.tasktools.org/scm/tm/timew.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert_equal version, shell_output("timew --version").strip
```

Comment:
Redunant according to [docs ("The version string")](http://www.rubydoc.info/github/Homebrew/brew/master/Formula#version-class_method)